app/services/workflows/main_workflow.rb

```diff
@@ -22,5 +24,10 @@ def version_5_verification(assessment)
       Utilities::ProceedingTypeThresholdPopulator.call(assessment)
       Creators::EligibilitiesCreator.call(assessment)
     end
+
+    def applicant_receives_relevant_asylum_support?(assessment)
+      assessment.proceeding_types.all? { _1.ccms_code.to_sym.in?(CFEConstants::IMMIGRATION_AND_ASYLUM_PROCEEDING_TYPE_CCMS_CODES) } &&
+        assessment.applicant.receives_asylum_support
+    end
```

Comment:
This seems a complex question. Seems like we should be calling a different workflow for each proceeding type - something we haven't done before. Then the existing combining rules should do 'the right thing'

Reply:
Maybe my mistake is to call the Asylum-Support-receiving pathway a workflow. Really what the logic should be saying is: "if there's no reason to perform a means assessment, don't perform the means assessment". The existence of a non A&I proceeding type, or the absence of asylum support, constitutes a reason to perform a means assessment. Otherwise we should just _not_ call a workflow, rather than call a new workflow. Hmm. Will ponder this one.